<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TimesSettingsWindow</class>
 <widget class="QWidget" name="TimesSettingsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>622</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <layout class="QGridLayout" name="layout">
     <item row="0" column="0">
      <widget class="QFrame" name="frame">
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QPushButton" name="back">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>370</y>
          <width>80</width>
          <height>23</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Back</string>
        </property>
       </widget>
       <widget class="QPushButton" name="next">
        <property name="geometry">
         <rect>
          <x>510</x>
          <y>370</y>
          <width>80</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Next</string>
        </property>
       </widget>
       <widget class="QLabel" name="title">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>601</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Times settings</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="description">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>581</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Select the time range you would like to use to save the input video.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
       </widget>
       <widget class="QLabel" name="errorEndTime">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>220</y>
          <width>581</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;The end time is invalid. It must follow the pattern hh:mm:ss:ff. Verify that it is in the range and that it is after the start time.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLabel" name="tooltip">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>450</x>
          <y>330</y>
          <width>141</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;You have errors. Please correct them.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPlainTextEdit" name="endTimeInput">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>265</y>
          <width>501</width>
          <height>31</height>
         </rect>
        </property>
        <property name="placeholderText">
         <string>Set the end time. Format: hh:mm:ss.ff</string>
        </property>
       </widget>
       <widget class="QLabel" name="properties">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>80</y>
          <width>581</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Properties</string>
        </property>
       </widget>
       <widget class="QLabel" name="inputFileLabel">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>100</y>
          <width>91</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Input file:</string>
        </property>
       </widget>
       <widget class="QLabel" name="lenthLabel">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>120</y>
          <width>91</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Length:</string>
        </property>
       </widget>
       <widget class="QLabel" name="inputFileValue">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>100</y>
          <width>471</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>[The input file]</string>
        </property>
       </widget>
       <widget class="QLabel" name="lengthValue">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>120</y>
          <width>471</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>[The total length of the video]</string>
        </property>
       </widget>
       <widget class="QLabel" name="errorStartTime">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>145</y>
          <width>581</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;The start time is invalid. It must follow the pattern hh:mm:ss.ff. Verify that it is in the range and that it is before the end time.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPlainTextEdit" name="startTimeInput">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>180</y>
          <width>501</width>
          <height>31</height>
         </rect>
        </property>
        <property name="placeholderText">
         <string>Set the start time. Format: hh:mm:ss.ff</string>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>185</y>
          <width>71</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Start time:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>270</y>
          <width>71</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>End time:</string>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
